vms/avm/network/config.go

```diff
@@ -21,9 +21,9 @@ var DefaultConfig = Config{
 	PushGossipMaxRegossipFrequency:              30 * time.Second,
 	PushGossipFrequency:                         500 * time.Millisecond,
 	PullGossipPollSize:                          1,
-	PullGossipFrequency:                         1500 * time.Millisecond,
```

Comment:
These constants were updated so we could conveniently use 3600 - but we could undo this diff.

In retrospect I'm also wondering if it makes sense to just decide the frequency we make requests based on the requests-per-validator throttling limit we configure...